docs/guides/admin/docs/workflowoperationhandlers/send-email-woh.md

```diff
@@ -277,3 +280,38 @@ Logged incident of the error looks like this:
   </#list>
 </#if>
 ```
+
+### Example 5
+
+The user name is stored in the episode dublin core `contributor` field. There's a user `jharvard` with email
+`jharvard@harvard.edu` defined in the system. The message will be sent to `jharvard@harvard.edu`:
+
+```xml
+   <operation
+      id="send-email"
+      fail-on-error="false"
+      description="Notify user associated to this recording that it is ready to be trimmed">
+      <configurations>
+        <configuration key="to">${(catalogs['episode']['contributor'])}</configuration>
+        <configuration key="subject">Recording is ready for EDIT</configuration>
+        <configuration key="body-template-file">eventDetails</configuration>
```

Comment:
Hi @rute-santos In case you have a good 'eventDetails' template at hands, it would be great if you could contribute it, too.